client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-setup/styles.scss

```diff
@@ -81,7 +47,13 @@ $font-family: 'SF Pro Text', $sans;
 			font-family: $font-family;
 			font-weight: 400;
 		}
+        input {
+            height: 44px;
+        }
+
```

Comment:
I don't think we need this. We can ignore the designs or change the original field because now only this step will have taller fields.

Reply:
Ignoring this for now. 👍